deepchecks/checks/performance/performance_report.py

```diff
@@ -97,14 +97,14 @@ def _performance_report(self, train_dataset: Dataset, test_dataset: Dataset, mod
         validate_model(test_dataset, model)
 
         task_type = task_type_check(model, train_dataset)
-        clasess = test_dataset.classes
+        clasess = train_dataset.classes
```

Comment:
why did you change to train?

Reply:
test can contain classes not in train

Reply:
missing labels from train should do an error either way no?

Reply:
not an error just bad score*